hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/MergeOnReadSnapshotRelation.scala

```diff
@@ -154,4 +159,7 @@ class MergeOnReadSnapshotRelation(val sqlContext: SQLContext,
     }).toList
     fileSplits
   }
+
+  // just for test
+  def getFileIndexPaths = fileIndex.map(x => x.dataFile.get.filePath)
```

Comment:
Is there any annotation like VisibleOnlyForTests as we have in java. Not a big fan of introducing public methods in source code just for test purposes.